test/helpers/api-unit.helper.js

```diff
@@ -70,3 +71,33 @@ export function generateHistory (days) {
 
   return history;
 }
+
+export function generateTodo (user) {
+  let todo = {
+    text: 'test todo',
+    type: 'todo',
+    value: 0,
+    completed: false,
+  };
+
+  let task = new Tasks.todo(Tasks.Task.sanitize(todo)); // eslint-disable-line babel/new-cap
+  task.userId = user._id;
+  task.save();
```

Comment:
well since you pass an object that contains tasks objects you can use it after the middleware has run. But i See the point with completed todos since those are not loaded